http/src/main/java/io/micronaut/http/filter/InternalFilter.java

```diff
@@ -2,35 +2,41 @@
 
 import io.micronaut.core.annotation.Internal;
 import io.micronaut.core.execution.ExecutionFlow;
+import io.micronaut.core.type.Executable;
 import io.micronaut.core.util.Toggleable;
+import io.micronaut.http.HttpRequest;
 import io.micronaut.http.HttpResponse;
 import io.micronaut.http.MutableHttpResponse;
-import io.micronaut.inject.ExecutableMethod;
 import org.reactivestreams.Publisher;
 
-import java.util.function.Supplier;
+import java.util.function.Function;
 
 @Internal
 public sealed interface InternalFilter {
     record Before<T>(
```

Comment:
Maybe you can call them Request/Response filters

Reply:
This is a name collision with the new annotations... but more descriptive. since theyre internal only they dont have to be all that descriptive though